### PR TITLE
Create helper API for Dashboard Components, Use app Store #1791

### LINKED ISSUE #1791: Create helper API for Dashboard Components, Remove jQuery Dependency
**Actual Behaviour**

Currently, API fetching is done using jquery ajax method for dashboard components. 

**Expected Behaviour**

Replace it with helper API

Component:

- MySkills.js
- MyRatings.js
- MyAnalytics.js

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1791

Changes:

- Remove jQuery dependency from Dashboard component, used helper API for fetching data
- Connect redux store and used app store instead of cookies
- Created Test for MyAnalytics
- Add redux store for tests for Dashboard component

Surge Deployment Link: https://pr-1853-1-fossasia-susi-skill-cms.surge.sh

Screenshots for the change:

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd pleas review it

File: src/components/Dashboard/MySkills.js [hunk 1]
```diff
@@ -268,4 +260,17 @@ class MySkills extends Component {
   }
 }
 
-export default MySkills;
+MySkills.propTypes = {
+  email: PropTypes.string,
+};
+
+function mapStateToProps(store) {
```
Comment (akshatnitd) [inline src/components/Dashboard/MySkills.js]: We use `state` here. Please maintain consistency

File: src/components/Dashboard/MySkills.js [hunk 2]
```diff
@@ -54,38 +53,33 @@ class MySkills extends Component {
...
-            skill_name: i.skill_name,
+            skillName: i.skillName,
             type: 'public',
             status: 'active',
             ...i,
           });
         }
-        self.setState({
+        this.setState({
           skillsData,
```
Comment (akshatnitd) [inline src/components/Dashboard/MySkills.js]: Do not maintain this info on the component state. Keep it on the App state.

**Comment by Orbiter** [comment]
please resolve conflicts

**Comment by fragm3** [comment]
Will resolve in a day or two

**Comment by fragm3** [comment]
@Orbiter @akshatnitd fixed the conflicts, please review

**Pull request merged**